src/1-run-ipfs-on-docker/index.md

```diff
@@ -0,0 +1,31 @@
+---
+baseurl: ..
+template: tmpl/layouts/post.html
+breadcrumbs:
+  - {name: "1-run-ipfs-on-docker", link: "./" }
+id: 1-run-ipfs-on-docker
+date: 2015-07-11
+title: Run IPFS in a Docker container
+author: Kyle Drake
+collection: posts
+---
+
+<a href="https://www.docker.com/">Docker</a> has become something of a standard for containerizing applications, even being used as the container standard for <a href="http://kubernetes.io/">Kubernetes</a>, Google's project to build an open source version of their internal clustering systems.
+
+There's a ridiculous amount of churn in this space, including recent work to <a href="http://runc.io/">develop an industry standard for containers</a> that will likely add even more churn. But it looks like the Docker format will be supported for quite some time, and whatever the new standard ends up being will probably look a lot like Docker.
+
+In my occasional role as a sysadmin, I definitely understand what containers bring to the table. I've configured many systems in the past, and then dealt with the frustration of figuring out how to configure them again, or quickly deploy and scale software to multiple servers, and make sure they don't consume too many resources and take systems down.
```

Comment:
Do we need to explain what Docker and containers are? I don't think this fits here. I think people will either be familiar with Docker, or not be the audience of this post. (i.e. if they don't know what docker is, they won't be able to use it to run IPFS, and it's out of scope to guide people through setting up Docker.) How about something like:

> In recent years, Docker and a few other projects have redefined how we run server applications. In the future, Containerized Applications could even improve how we run apps in our personal devices. At its core, this fast-paced improvement is a combination of good interfaces to standardize how to do things, and great tooling to make using containers easy.
>
> The IPFS Project has many things planned for the world of containers. The most interesting is using IPFS to distribute containers hyper efficiently across data-centers and the internet. We will be discussing many of these things in upcoming posts, but first things first. This post is a quick guide for running an IPFS node directly within Docker.